pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -120,6 +168,22 @@ func toRuntimeProtocol(protocol api.Protocol) runtimeApi.Protocol {
 	return runtimeApi.Protocol_TCP
 }
 
+// toKubeContainer converts runtimeApi.Container to kubecontainer.Container.
+func (m *kubeGenericRuntimeManager) toKubeContainer(c *runtimeApi.Container) (*kubecontainer.Container, error) {
+	if c == nil || c.Id == nil || c.Name == nil || c.Image == nil || c.State == nil {
```

Comment:
OK, the protobuf says they are optional, but in real case, we all assume they are not empty, right? Otherwise there is something wrong in our code.